### PR TITLE
Testing branch

### PR DESCRIPTION
**Pull request by CoderKevinZhang**
**What type of PR is this?**
> /kind test

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: globalscheduler/test/yaml/sample_30_clusters.yaml [hunk 1]
```diff
@@ -0,0 +1,962 @@
...
+    totalcapacity: 10
+  - flavorid: medium
+    totalcapacity: 20
+  geolocation:
+    area: East
+    city: NewYork
+    country: US
+    province: NewYork
+  homescheduler: scheduler2
+  ipaddress: 10.0.0.1
```
Comment (cathyhongzhang) [inline globalscheduler/test/yaml/sample_30_clusters.yaml:20]: you need to change the ip to the openstack cluster ip

File: globalscheduler/test/yaml/sample_30_clusters.yaml [hunk 2]
```diff
@@ -0,0 +1,962 @@
...
+  - flavorid: small
+    totalcapacity: 10
+  - flavorid: medium
+    totalcapacity: 20
+  geolocation:
+    area: East
+    city: NewYork
+    country: US
+    province: NewYork
+  homescheduler: scheduler2
```
Comment (cathyhongzhang) [inline globalscheduler/test/yaml/sample_30_clusters.yaml:19]: do not set the homescheduler field value to any scheduler

File: globalscheduler/test/yaml/sample_dispatcher.yaml [hunk 3]
```diff
@@ -1,6 +1,13 @@
...
 spec:
-  cluster: ["c1", "c2"]
+  cluster: [""]
+---
+apiVersion: globalscheduler.com/v1
+kind: Dispatcher
+metadata:
+  name: dispatcher2
+spec:
+  cluster: [""]
```
Comment (cathyhongzhang) [inline globalscheduler/test/yaml/sample_dispatcher.yaml:13]: please check the dispatcher spec after merge of PR 157 and update the fields accordingly

File: globalscheduler/test/yaml/sample_scheduler.yaml [hunk 4]
```diff
@@ -42,3 +42,93 @@ spec:
     cpucapacity: [200,201]
     memcapacity: [300,301]
     serverprice: [400,401]
+---
+apiVersion: globalscheduler.com/v1
+kind: Scheduler
+metadata:
+  name: orange-2
+spec:
```
Comment (cathyhongzhang) [inline globalscheduler/test/yaml/sample_scheduler.yaml:50]: update the yaml to just include needed info.

**Review by cathyhongzhang — APPROVED** [review]
LGTM

**Review by kimeunju108 — APPROVED** [review]
LGTM

**Pull request merged**